AssertionError: `child` is a required argument.

Hi,

I'm getting an assertion error after including the app to my project following the instructions.
The relevant traceback is as follows: 

```
    class TemplateSerializer(TaggitSerializer,serializers.ModelSerializer):
    File "/app/contratos/api/documents/serializers.py", line 8, in TemplateSerializer
    tags = TagListSerializerField()
    File "/usr/local/lib/python2.7/site-packages/rest_framework/fields.py", line 1126, in __init__
    assert self.child is not None, '`child` is a required argument.
    AssertionError: `child` is a required argument
```

 As reference take into account

```
     $uname -a

    Darwin Juans-MacBook-Pro.local 13.4.0 Darwin Kernel Version 13.4.0: Wed Dec 17 19:05:52 PST 2014; root:xnu-2422.115.10~1/RELEASE_X86_64 x86_64
```

requirements:

```
    Django==1.7.6
    django-taggit==0.12.3
    django-taggit-serializer==0.1.0
    djangorestframework==3.0.3
```

Everything is running inside boot2docker postgres 9.4

regards,

--- a/taggit_serializer/serializers.py
+++ b/taggit_serializer/serializers.py
@@ -3,6 +3,7 @@ from rest_framework import serializers
 
 
 class TagListSerializerField(serializers.ListField):
+    child = serializers.CharField()
 
     def to_representation(self, obj):
         if type(obj) is not list:

